SQL/paradise_schema.sql

```diff
@@ -285,6 +285,7 @@ CREATE TABLE `player` (
   `screentip_color` varchar(7) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '#ffd391',
   `ghost_darkness_level` tinyint(1) UNSIGNED NOT NULL DEFAULT '255',
   `colourblind_mode` VARCHAR(48) NOT NULL DEFAULT 'None' COLLATE 'utf8mb4_general_ci',
+  `keybindings` longtext COLLATE utf8mb4_unicode_ci DEFAULT NULL,
```

Comment:
```suggestion
  `keybindings` LONGTEXT COLLATE utf8mb4_unicode_ci DEFAULT NULL,
```

Reply:
are you sure? All other instances of longtext in the sql files are lowercase

Reply:
Yes. All SQL datatypes are `ALLCAPS` 
![image](https://user-images.githubusercontent.com/25063394/176884299-e3beea39-8bbf-4e69-a933-fd1737f00f9b.png)